packages/@rnw-scripts/create-github-releases/package.json

```diff
@@ -24,6 +24,7 @@
     "lodash": "^4.17.15",
     "node-fetch": "^2.6.0",
     "semver": "^7.3.2",
+    "simple-git": "^1.131.0",
```

Comment:
Not actually related, but fixing this dep while we're here for usage outside of monorepo.